networking/amplify/backend/api/mobile/schema.graphql

```diff
@@ -29,9 +29,11 @@ type GameTemplate @model{
   phaseOneTime: Int!
   phaseTwoTime: Int!
   imageUrl: String
-  questionTemplates: [QuestionTemplate] @manyToMany(relationName: "GameQuestions")
+  questionTemplates: [QuestionTemplate] @manyToMany(relationName: "GameQuestions") 
+  numQuestionTemplates: Int! @default(value: "0")
   createdAt: AWSDateTime
   updatedAt: AWSDateTime
+  type: String @default(value: "GameTemplate") @index(name: "gameTemplatesByDate", queryField: "gameTemplatesByDate", sortKeyFields: ["updatedAt"]) @index(name: "gameTemplatesByGrade", queryField: "gameTemplatesByGrade", sortKeyFields: ["grade"]) @index(name: "gameTemplatesByNumQuestionTemplates", queryField: "gameTemplatesByNumQuestionTemplates", sortKeyFields: ["numQuestionTemplates"])
 }
```

Comment:
these are the indexes that allow us to sort by the various fields. Annoying we can't use `__typename` and have to instead add in a redundant `type` field but this is per `Amplify` guidance as outlined here: 

https://github.com/aws-amplify/amplify-category-api/issues/45